pkg/services/featuremgmt/static_evaluator.go

```diff
@@ -36,7 +36,7 @@ func ProvideStaticEvaluator(cfg *setting.Cfg) (StaticFlagEvaluator, error) {
 
 // createStaticEvaluator evaluator that allows evaluating static flags from config.ini
 func createStaticEvaluator(providerType string, u *url.URL, staticFlags map[string]bool) (StaticFlagEvaluator, error) {
-	provider, err := createProvider(providerType, u, staticFlags)
+	provider, err := createProvider("", providerType, u, staticFlags, nil, nil)
```

Comment:
why'd `providerType` get replaced with `""`?

Reply:
oh, nm, I see now there was a parameter added at the beginning...

Reply:
yep, feel free to reorder params to your liking